.vscode/launch.json

```diff
@@ -21,11 +21,16 @@
       "outFiles": [ "${workspaceRoot}/compiled/**/*.js" ]
     },
     {
-      "name": "Attache Remote App.js",
+      "name": "Launch Remote App",
       "type": "node",
+      "protocol": "inspector",
       "request": "attach",
+      "stopOnEntry": false,
       "port": 10000,
-      "address": "192.168.10.10"
-    },
+      "address": "192.168.10.10",
```

Comment:
So does this mean anyone using this in-house will need to remember to update this address to their own internal IP?